test/helper.ts

```diff
@@ -33,3 +35,17 @@ export function parseAndProcess(input: string, fn: (ast: ESTree.Program) => void
   fn(ast);
   return astring(ast).trim();
 }
+
+const defaultSandbox = {
+  console,
+  module: {
+    exports: {}
+  },
+  require
+};
+
+export function virtualModule(code: string, optionsSandbox = {}) {
```

Comment:
Basically optionsSandbox allows for injection of arbitrary stuff into `vms` sandbox, so I don't see what should/could be typed here.